### PR TITLE
various tweaks and changes

### PR DESCRIPTION
**Pull request by jcooper-korg**

### CODE REVIEW

File: misc/basic_keys.py [hunk 1]
```diff
@@ -2,7 +2,12 @@
 import string
 from ..utils import normalise_keys, insert
 
-alpha_alt = "air bat cap drum each fine gust harp sit jury crunch look made near odd pit quench red sun trap urge vest whale plex yank zip".split()
+# Talon recommended alphabet
+# alpha_alt = "air bat cap drum each fine gust harp sit jury crunch look made near odd pit quench red sun trap urge vest whale plex yank zip".split()
```
[the remark this answers is not in the document]
Comment (jcooper-korg) [reply misc/basic_keys.py]: Doh- yes of course it should! I did that in my initial change in 2e4f61a4, but inadvertently checked in my local mod in the unrelated commit 562b0cd1.  I guess we should pursue a config.json change after all on this-- too easy to accidentally check in a local tweak.

File: misc/basic_keys.py [hunk 2]
```diff
@@ -116,7 +121,7 @@ def press_keys(m):
 ctx = Context("basic_keys")
 ctx.keymap(
     {
-        "(uppercase | ship | sky) {basic_keys.alphabet}+ [(lowercase | sunk)]": uppercase_letters,
+        "(uppercase | ship | sky) {basic_keys.alphabet}+ [(lowercase | lower | sunk)]": uppercase_letters, # jsc added lower
```
[the remark this answers is not in the document]
Comment (jcooper-korg) [reply misc/basic_keys.py]: got it. I've been removing my 'jsc' notes elsewhere as I go, but this was left over (pasted in) from my old changes from before i was participating in talon_community.

File: misc/basic_keys.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (jcooper-korg) [reply misc/basic_keys.py]: commit 84399278 reverts the entire commit 562b0cd1 which also included an failed experiment with option+drag .   commit 607d58e5 brings back just the change to window_control

File: misc/basic_keys.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (jcooper-korg) [reply misc/basic_keys.py]: commit 3be80f64 removes this comment.

**Comment by dwiel** [comment]
thanks for all of the additions!

**Pull request merged**